pkg/auth/iguazio/abstract.go

```diff
@@ -39,6 +40,7 @@ const (
 type AbstractAuth struct {
 	Logger     logger.Logger
 	HttpClient *http.Client
+	Cache      *cache.LRUExpireCache
```

Comment:
This is not used in v4, is it her just in case?

Reply:
@TomerShor a caching was discussed on HLD, so I decided to prepare the ground for it and implement the caching is separate pr